src/libs/EmojiUtils.js

```diff
@@ -89,20 +89,40 @@ function getDynamicHeaderIndices(emojis) {
 /**
  * Get number of empty spaces to be filled to get equal emojis for every row
  * @param {Number} emojiCount
+ * @param {Number} suffix
  * @returns {Object[]}
  */
-function getDynamicSpacing(emojiCount) {
+function getDynamicSpacing(emojiCount, suffix) {
     const spacerEmojis = [];
     let modLength = CONST.EMOJI_NUM_PER_ROW - (emojiCount % CONST.EMOJI_NUM_PER_ROW);
     while (modLength > 0) {
         spacerEmojis.push({
-            code: CONST.EMOJI_SPACER,
+            code: `${CONST.EMOJI_SPACER}_${suffix}_${modLength}`,
```

Comment:
What is the purpose of changing the code like this?

Reply:
I am getting an error when in `Encountered two children with the same key` with the `EMOJI_SPACER`. Because in the `keyExtractor={item => 'emoji_picker_${item.code}'}`

Reply:
Ah gotcha and now instead of using the code for spacers, you just check for `spacer: true`. Sweet